package/MDAnalysis/transformations/fit.py

```diff
@@ -0,0 +1,206 @@
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""\
+Fitting transformations --- :mod:`MDAnalysis.transformations.fit`
+=================================================================
+
+Translate and/or rotates the coordinates of a given trajectory to align
+a given AtomGroup to a reference structure.
+
+.. autofunction:: fit_translation
+
+.. autofunction:: fit_rot_trans
+
+"""
+from __future__ import absolute_import
+
+import numpy as np
+from functools import partial
+
+from ..analysis import align
+from ..lib.util import get_weights
+from ..lib.transformations import euler_from_matrix, euler_matrix
+
+
+def fit_translation(ag, reference, plane=None, weights=None):
+
+    """Translates a given AtomGroup so that its center of geometry/mass matches 
+    the respective center of the given reference. A plane can be given by the
+    user using the option `plane`, and will result in the removal of
+    the translation motions of the AtomGroup over that particular plane.
+    
+    Example
+    -------
+    Removing the translations of a given AtomGroup `ag` on the XY plane by fitting 
+    its center of mass to the center of mass of a reference `ref`:
+    
+    .. code-block:: python
+    
+        ag = u.select_atoms("protein")
+        ref = mda.Universe("reference.pdb")
+        transform = mda.transformations.fit_translation(ag, ref, plane="xy",
+                                                        center_of="mass")
+        u.trajectory.add_transformations(transform)
+    
+    Parameters
+    ----------
+    ag : Universe or AtomGroup
+       structure to translate, a
+       :class:`~MDAnalysis.core.groups.AtomGroup` or a whole 
+       :class:`~MDAnalysis.core.universe.Universe`
+    reference : Universe or AtomGroup
+       reference structure, a :class:`~MDAnalysis.core.groups.AtomGroup` or a whole 
+       :class:`~MDAnalysis.core.universe.Universe`
+    plane: str, optional
+        used to define the plane on which the translations will be removed. Defined as a 
+        string of the plane. Suported planes are yz, xz and xy planes.
+    weights : {"mass", ``None``} or array_like, optional
+       choose weights. With ``"mass"`` uses masses as weights; with ``None``
+       weigh each atom equally. If a float array of the same length as
+       `ag` is provided, use each element of the `array_like` as a
+       weight for the corresponding atom in `ag`.
+    
+    Returns
+    -------
+    MDAnalysis.coordinates.base.Timestep
+    """
+    
+    if plane is not None:
+        axes = {'yz' : 0, 'xz' : 1, 'xy' : 2}
+        try:
+            plane = axes[plane]
+        except (TypeError, KeyError):
+            raise ValueError('{} is not a valid plane'.format(plane))
+    try:
+        if ag.atoms.n_residues != reference.atoms.n_residues:
+            raise ValueError("{} and {} have mismatched number of residues".format(ag,reference))
```

Comment:
This is the only line of code (raising the `ValueError`) in the new module that isn't covered by tests. So, 99 % coverage, which is excellent.

Reply:
My local work suggests we'll be up to 100 % once CI runs again -- if so, this can be marked as resolved.